ui-validation/src/test/kotlin/edu/stanford/spezi/ui/validation/SpeziValidationTest.kt

```diff
@@ -1,13 +1,17 @@
 package edu.stanford.spezi.ui.validation
 
 import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.GlobalScope
 import org.junit.Test
 
 class SpeziValidationTest {
 
     @Test
     fun testValidationDebounce() {
-        val engine = ValidationEngineImpl(rules = listOf(ValidationRule.nonEmpty))
+        val engine = ValidationEngineImpl(
+            rules = listOf(ValidationRule.nonEmpty),
+            coroutineScope = GlobalScope
```

Comment:
We should use test scopes for tests, consider SpeziTestScope or TestScope directly

Reply:
Sure, didn't know about that - sounds good!